docs/geometryApproximation.md

```diff
@@ -0,0 +1,43 @@
+# Geometry Approximation example
+
+![Geometry Approximation Example Image](https://topology-tool-kit.github.io/img/gallery/geometryApproximation.jpg)
+
+## Pipeline description
+This example loads two geometric structures: the streamlines of a section of ground water and its surrounding stone.
```

Comment:
I don't believe the python script needs to load the streamlines (see my comments below)